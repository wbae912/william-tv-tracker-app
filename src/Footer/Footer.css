.footer {
  color: white;
  font-family: 'Montserrat', sans-serif;
  background-color: rgb(78, 66, 61);
  height: 40px;
  display: flex;
  justify-content: center;
  align-items: center;
  margin-top: 20px;
}

.footer-p {
  font-size: 12px;
}