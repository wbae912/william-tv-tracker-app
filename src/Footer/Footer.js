import React, { Component } from 'react';
import './Footer.css';

export default class Footer extends Component {
  render() {
    return (
      <footer className="footer">
        <p className="footer-p">Created by William Bae</p>
      </footer>
    )
  }
}
