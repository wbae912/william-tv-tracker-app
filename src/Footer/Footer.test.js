import React from 'react';
import ReactDOM from 'react-dom';
import Footer from './Footer';
import { MemoryRouter } from 'react-router';
import renderer from 'react-test-renderer';

describe('AppNav component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<MemoryRouter><Footer /></MemoryRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  
  it('renders UI as expected', () => {
    const tree = renderer
      .create(<MemoryRouter><Footer /></MemoryRouter>)
      .toJSON();
      expect(tree).toMatchSnapshot();
  })
})