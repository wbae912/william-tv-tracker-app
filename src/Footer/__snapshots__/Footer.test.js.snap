// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppNav component renders UI as expected 1`] = `
<footer
  className="footer"
>
  <p
    className="footer-p"
  >
    Created by William Bae
  </p>
</footer>
`;
